libs/ssl-config/build.gradle

```diff
@@ -10,6 +10,7 @@ apply plugin: "elasticsearch.publish"
 
 dependencies {
   api project(':libs:core')
+  api project(':libs:entitlement')
```

Comment:
This and the module-info changes are needed to access `NotEntitledException` in the relevant modules